frontend/packages/client/src/hooks/useUserRoleOnCommunity.js

```diff
@@ -1,27 +1,35 @@
+import { useCallback, useEffect, useState } from "react";
+import { isEmptyArray } from "utils";
 import useUserCommunities from "./useUserCommunities";
 
 export default function useUserRoleOnCommunity({
   addr,
   communityId,
   roles = [],
 } = {}) {
-  const { data: communityUser, loading } = useUserCommunities({
-    addr,
+
+  const [hasRole, setHasRole] = useState();
```

Comment:
this is an extra state that is not necessary, the hook `useUserCommunities` provides a reusable state that holds the data we need to pass as a result of the `useUserRoleOnCommunity` and the useUserCommunities already provides loading state and it triggers it self so the useEffect wouldn't be necessary

Reply:
That is true... I'm not sure where I got caught in an intermediate step, but reverting these changes after confirming everything works without them. Thanks!